### ISSUE TITLE
W3C DeviceOrientation Event Specification is no longer in active maintenance

### ISSUE THREAD

**Issue by traildex** [apache/cordova-plugin-device-orientation#44]
> With the W3C Device Orientation API now being supported on iOS, Android and Windows devices, this plugin is not needed any more.

Cordova links the above to a 2016 W3C standard which pops up
> This version is outdated!

Redirecting to...
[W3C](https://www.w3.org/TR/orientation-event/)
> Beware. This specification is no longer in active maintenance and the Geolocation Working Group does not intend to maintain it further.

Hopefully I'm missing something. It "seems" Cordova has depreciated the device-orientation plugin AND W3C does not intend to maintain the DeviceOrientation Event Specification.

**Comment by janpio** [comment]
Does the code indicated as a migration in https://blog.phonegap.com/migrating-from-the-cordova-device-orientation-plugin-8442b869e6cc (linked from the deprecation notice) still work?

**Comment by janpio** [comment]
Yeah, problem is that W3C moved on, bu browser still implement it. As there is not alternative from W3C (that I am aware of), this currently is as good as it gets :/
